### DISCUSSION TITLE
that was an issue related to the project

### DISCUSSION THREAD

**Discussion by Wrenn96** [Berny23/LD-ToyPad-Emulator#163] [General]
no need to reply to a closed issue aswell

**Comment by Euniemeansme** [comment]
1. You closed the issue without even allowing someone to try to help you with your issue
2. Your issue was being caused by an external program, not by the toypad emulator that was built here.

**Comment by Wrenn96** [comment]
i closed the issue because i dropped it and just used the site on the emulator. However the installation says that I can copy and paste the code, but that is under the assumption that I have already opened the site on the VM.

**Comment by cort1237** [comment]
It's not relevant because its entirely dependent on the VM you chose to use, the VM's settings, and as you discovered on your own even if the clipboard isnt shared you can still copy the code within the emulator.

And finally, making an issue just to immediately close is just not the purpose of issues. If you discover a solution to your issue, you should share it. This is a collaborative environment. So yes I did need to reply to a closed issue so that anyone in the future can look at that and see what they need to do.
